app/controllers/comments_controller.rb

```diff
@@ -2,12 +2,25 @@
 class CommentsController < ApplicationController
   before_action :authenticate_user!
 
+  def index
+    return redirect_back(fallback_location: root_path) unless request.xhr?
+
+    @comments = @commentable.comments.ordered.page(params[:page]).per(3)
```

Comment:
per(3) is it ok? if not change before merge, not sure if ok to show only 3 comments for pagination, maybe 10?